pkg/services/sqlstore/migrator/dialect.go

```diff
@@ -368,7 +368,7 @@ func (b *BaseDialect) InsertQuery(tableName string, row map[string]any) (string,
 	for col := range row {
 		keys = append(keys, col)
 	}
-	slices.Sort[string](keys)
+	slices.Sort(keys)
```

Comment:
unrelated, but seems OK 🤷🏻

Reply:
Some go.mod upgrades that had this side-effect.